source/_components/notify.html5.markdown

```diff
@@ -18,6 +18,10 @@ The `html5` notification platform enables you to receive push notifications to C
 HTML5 push notifications **do not** work on iOS.
 </p>
 
+<p class='note warning'>
+The GCM configuration option is deprecated and will stop working in April 2019, see [https://developers.google.com/cloud-messaging/faq](https://developers.google.com/cloud-messaging/faq). If you are installing this platform for the first time, follow the VAPID configuration steps. To migrate your current installation from GCM to VAPID configuration, follow the instructions below. You can skip the first 3 steps and continue in step 4 with your existing project. You will also need to [re-enable the notifications in your browser](#setting-up-your-browser).
```

Comment:
For urls you must use html code 😉

Reply:
Hmm ok, but for my understanding: why is that? 
The new build looks and works fine: https://deploy-preview-9019--home-assistant-docs.netlify.com/components/notify.html5/

Reply:
well if it works, it works 😄 Apparently this has improved since the last netifly update.

You can ignore my first comment.